bin/pulsar

```diff
@@ -44,12 +44,18 @@ FUNCTIONS_EXTRA_DEPS_DIR=${PULSAR_FUNCTIONS_EXTRA_DEPS_DIR:-"${DEFAULT_FUNCTIONS
 SQL_HOME=$PULSAR_HOME/pulsar-sql
 PRESTO_HOME=${PULSAR_HOME}/lib/presto
 
-
+# Check pulsar env and load pulser_env.sh
 if [ -f "$PULSAR_HOME/conf/pulsar_env.sh" ]
 then
     . "$PULSAR_HOME/conf/pulsar_env.sh"
 fi
 
+# Check bookkeeper env and load bkenv.sh
```

Comment:
Move this before line 47 for backward compatibility consideration.
So if people already define environments in pulsar_env, those env vars will not be overridden by bkenv.sh.